layouts/cover.vue

```diff
@@ -48,4 +49,10 @@ const props = defineProps({
         @apply h-full drop-shadow-lg
     }
 }
+
+.placeholder {
```

Comment:
```suggestion
.shape-placeholder {
```

What is this a placeholder for?